src/EnergyPlus/Psychrometrics.hh

```diff
@@ -105,58 +105,57 @@ namespace Psychrometrics {
-
-    extern Array1D_bool const PsyReportIt; // PsyTdpFnTdbTwbPb     1 | PsyRhFnTdbWPb        2 | PsyTwbFnTdbWPb       3 | PsyVFnTdbWPb         4 |
-                                           // PsyWFnTdpPb          5 | PsyWFnTdbH           6 | PsyWFnTdbTwbPb       7 | PsyWFnTdbRhPb        8 |
-                                           // PsyPsatFnTemp        9 | PsyTsatFnHPb         10 | PsyTsatFnPb          11 | PsyRhFnTdbRhov       12 |
-                                           // PsyRhFnTdbRhovLBnd0C 13 | PsyTwbFnTdbWPb       14 - HR | PsyTwbFnTdbWPb       15 - max iter |
-                                           // PsyWFnTdbTwbPb       16 - HR | PsyTsatFnPb          17 - max iter | PsyTwbFnTdbWPb_cache 18 -
-                                           // PsyTwbFnTdbWPb_raw (raw calc) | PsyPsatFnTemp_cache  19 - PsyPsatFnTemp_raw (raw calc)
-#endif
-
-#ifndef EP_psych_errors
-#endif
+    constexpr std::array<std::string_view, static_cast<int>(PsychrometricFunction::Num)> PsyRoutineNames = {
+        "PsyTdpFnTdbTwbPb",
+        "PsyRhFnTdbWPb",
+        "PsyTwbFnTdbWPb",
+        "PsyVFnTdbWPb",
+        "PsyWFnTdpPb",
+        "PsyWFnTdbH",
+        "PsyWFnTdbTwbPb",
+        "PsyWFnTdbRhPb",
+        "PsyPsatFnTemp",
+        "PsyTsatFnHPb",
+        "PsyTsatFnPb",
+        "PsyRhFnTdbRhov",
+        "PsyRhFnTdbRhovLBnd0C",
+        "PsyTwbFnTdbWPb",
+        "PsyTwbFnTdbWPb",
+        "PsyWFnTdbTwbPb",
+        "PsyTsatFnPb",
+        "PsyTwbFnTdbWPb_cache",
+        "PsyPsatFnTemp_cache"}; // 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 |
+                                // 14 - HR | 15 - max iter | 16 - HR | 17 - max iter | 18 -
+                                // PsyTwbFnTdbWPb_raw (raw calc) | 19 - PsyPsatFnTemp_raw
+                                // (raw calc)
 
-#ifdef EP_cache_PsyTwbFnTdbWPb
-    constexpr int twbcache_size = 1024 * 1024;
-    constexpr int twbprecision_bits = 20;
-#endif
-#ifdef EP_cache_PsyPsatFnTemp
-    constexpr int psatcache_size = 1024 * 1024;
-    constexpr int psatprecision_bits = 24; // 28  //24  //32
-    constexpr Int64 psatcache_mask = psatcache_size - 1;
-#endif
-#ifdef EP_cache_PsyTsatFnPb
-    constexpr int tsatcache_size = 1024 * 1024;
-    constexpr int tsatprecision_bits = 24;
-    constexpr Int64 tsatcache_mask = tsatcache_size - 1;
-#endif
-#ifdef EP_cache_PsyTsatFnHPb
-    constexpr int tsat_hbp_cache_size = 1024 * 1024;
-    constexpr int tsat_hbp_precision_bits = 28;
+#ifdef EP_psych_stats
+    constexpr std::array<bool, static_cast<int>(PsychrometricFunction::Num)> PsyReportIt = {
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        false,
+        false,
+        false,
+        false,
+        true,
+        true}; // PsyTdpFnTdbTwbPb     1 | PsyRhFnTdbWPb        2 | PsyTwbFnTdbWPb       3 | PsyVFnTdbWPb         4 |
```

Comment:
Same with this comment block.  On this one it is probably worth pulling each one up to its associated element for clarity.